crates/goose-cli/src/session.rs

```diff
@@ -107,6 +107,9 @@ impl<'a> Session<'a> {
             match input.input_type {
                 InputType::Message => {
                     if let Some(content) = &input.content {
+                        if content.trim().is_empty() {
+                            continue;
+                        }
```

Comment:
I think it would be better to keep session.rs clear of any content parsing. Putting this logic in rustyline.rs where other user interaction elements are and returning. 'AskAgain' for this scenario is better I think.

aside: We should probably just delete cliclack.rs at this stage.

Reply:
Aha, yes, that seems better